src/app/containers/SocialEmbed/Cps/index.test.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import loggerMock from '#testHelpers/loggerMock';
+import { SOCIAL_EMBED_RENDERED } from '#lib/logger.const';
+import CpsSocialEmbedContainer from '.';
+import withContexts from '../common/testHelper';
+import { cpsTwitterBlock, cpsTwitterBlockNoEmbed } from '../common/fixtures';
+
+/* eslint-disable react/prop-types */
+jest.mock('react-lazyload', () => {
+  return function MockedLazyload({ children }) {
+    return <>{children}</>;
+  };
+});
```

Comment:
I think this is equivalent?
```suggestion
jest.mock('react-lazyload', () => React.Fragment)
```

Reply:
Our tests say no. 😂

Reply:
OK looks like it's not 😝